junebug/stores.py

```diff
@@ -41,21 +47,37 @@ def load_property(self, id, key):
 
 
 class InboundMessageStore(BaseStore):
+    KEYSPACE = 'inbound_messages'
     '''Stores the entire inbound message, in order to later construct
     replies'''
-    def _get_key(self, channel_id, message_id):
-        return '%s:inbound_messages:%s' % (channel_id, message_id)
-
     def store_vumi_message(self, channel_id, message):
         '''Stores the given vumi message'''
-        key = self._get_key(channel_id, message.get('message_id'))
+        key = self.get_key(channel_id, message.get('message_id'))
         return self.store_property(key, 'message', message.to_json())
 
     @inlineCallbacks
     def load_vumi_message(self, channel_id, message_id):
         '''Retrieves the stored vumi message, given its unique id'''
-        key = self._get_key(channel_id, message_id)
+        key = self.get_key(channel_id, message_id)
         msg_json = yield self.load_property(key, 'message')
         if msg_json is None:
             returnValue(None)
         returnValue(TransportUserMessage.from_json(msg_json))
+
+
+class OutboundMessageStore(BaseStore):
+    KEYSPACE = 'outbound_messages'
+    '''Stores the event url, in order to look it up when deciding where events
+    should go'''
+    def store_vumi_message(self, channel_id, api_request, message):
```

Comment:
The idea I had doing it this way is that we might want to store other things in the future, and then we wouldn't need to change all the places that this is called, because all the information is here already (say if we decided we also wanted to store the content). Then we would just change the call to a `store_all` with a dictionary, and we keep it to only one redis call.

We could change it to just store the event url, if you think that's a better solution for now.